src/app/garfile/arrival/index.njk

```diff
@@ -50,7 +50,7 @@
                   <input class="govuk-input govuk-date-input__input govuk-input--width-2" id="arrivalDay"
                     name="arrivalDay" type="number" pattern="[0-9]*" min="1" max="31"
                     value="{{cookie.dateSlice('day', cookie.getGarArrivalVoyage().arrivalDate)}}"
-                    aria-describedby="arrivalDate-error">
+                    {{ 'aria-describedby=arrivalDate-error' if errors | containsError('arrivalDate') }}>
```

Comment:
WAVE raised an error if 'arrivalDate-error' was set if there wasn't an actual span with id="arrivalDate-error" so this nunjuck filter goes through the error array and checks whether there is an element with identifier = 'arrivalDate' and sets the attribute if true.

This is pretty much everywhere where aria-describedby and errors are involved.